### PR TITLE
feat(admin): e2e login command

### PR DESCRIPTION
**Pull request by sarkapalkovicova**
Added and used new custom command for login.

### CODE REVIEW

**Review by mattjoke — CHANGES_REQUESTED** [review]
I'm unsure about that line I mentioned in the comment before. Otherwise, the code looks good.

File: apps/admin-gui-e2e/src/support/commands.ts [hunk 1]
```diff
@@ -23,3 +23,13 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('login', (envRole: string, baPrincipalName: string) => {
+  envRole = envRole ? '_' + envRole : envRole;
+
+  sessionStorage.setItem('baPrincipal', `{"name": "${baPrincipalName}"`);
+  sessionStorage.setItem('basicUsername', Cypress.env('BA_USERNAME' + envRole) as string);
```
Comment (HejdaJakub) [inline apps/admin-gui-e2e/src/support/commands.ts:31]: It should be achievable like this (the same also for the password).
```suggestion
  sessionStorage.setItem('basicUsername', String(Cypress.env('BA_USERNAME' + envRole)));
```

**Review by mattjoke — APPROVED** [review]
Looks good 👍

**Review by xflord — APPROVED** [review]
well done

**Pull request merged**